packages/api/app/routes.js

```diff
@@ -36,6 +37,9 @@ if (process.env.FAKE_AUTH === 'true') {
   ensureAuthorized = token.ensureAuthorized;
 }
 
+// dead letters endpoint
+router.use('/deadLetters', ensureAuthorized, deadLetters);
```

Comment:
Thoughts on the endpoint naming?

Reply:
"There are two hard problems in computer science..." 

Really what this endpoint is doing re-running the database update for a cumulus message.  

/messageWrite ?  /messageRecover?

Reply:
Maybe something with a subpath like `/cumulusMessage/recover`?

Reply:
I would suggest a subpath of `workflowMessages`. So `/deadLetters/workflowMessages`. It seems pretty clear then what type of data the endpoint is handling. You could go `/deadLetters/workflowMessages/process`, but not sure if it's not necessary

Reply:
Maybe `/deadLetterArchive/recoverCumulusMessages`

Reply:
https://github.com/nasa/cumulus/pull/2181/commits/9f8fdf63e3c3cc0866389dae6ecfbb25a7ff00d2